xdebug_com.c

```diff
@@ -485,17 +485,20 @@ int xdebug_is_debug_connection_active()
 
 int xdebug_is_debug_connection_active_for_current_pid()
 {
+	/* Early return to save some getpid() calls */
+	if (!xdebug_is_debug_connection_active()) {
+		return 0;
+	}
+
```

Comment:
C90 doesn't allow mixing declarations and code, and hence, you can't have the ``long pid = getpid();`` below, after the code above. Instead, you need to have the ``long pid;`` at the start of the function, and ``pid = getpid()`` where you currently have ``long pid = getpid();``.

Reply:
Fixed, but the AppVeyor failures puzzles me.